js/dispatch_flatbuffers.ts

```diff
@@ -0,0 +1,151 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import * as flatbuffers from "./flatbuffers";
+import { DenoError } from "./errors";
+import { core } from "./core";
+import * as msg from "gen/cli/msg_generated";
+import * as util from "./util";
+import { OP_FLATBUFFER } from "./dispatch";
+export { msg, flatbuffers };
```

Comment:
I suspect [the bundle size increase](https://github.com/denoland/deno/pull/2796#issuecomment-523980998) is a result of this line.